design/common.blocks/modal/_theme/modal_theme_islands.styl

```diff
@@ -3,7 +3,9 @@
     display: none;
     visibility: hidden;
 
-    background: rgba(50, 50, 50, .2);
+    margin: -9999px 0 0 -9999px;
```

Comment:
I think, we should add comment here, because it was not obviously when i dig this hack in `.popup` :]
Something like "to prevent clickability and visiblity of internal elements  with visiblity: visible" will be ok)